modules/problem_1/main.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by alyosharomanov on 10/28/2024.
+//
+#pragma once
+
+#include <vector>
+
+class Solution {
+public:
+    // Calculate the largest rectangle area that can be formed using a list of heights.
```

Comment:
I am confused. Is this a comment for the other function?
Don't use `list` for a vector.